recipes/cute_headers/all/conanfile.py

```diff
@@ -0,0 +1,40 @@
+from conans import ConanFile, tools
+import os
+import glob
+
+
+class CuteHeadersConan(ConanFile):
+    name = "cute_headers"
```

Comment:
Do we want it to be `cute_headers` or `cute-headers`?
The name of the github project is `cute_headers`.
vcpkg has named it `cute-headers`.

I vote for `cute-headers`.

Reply:
I named it `cute_headers` on purpose.

First of all `def source(self):` is easier to implement when the name is the same as the name of github project.
Second, it is always cute_something in the repo, almost all the files.
And the last reason - there's no `cute-headers` anywhere except vcpkg. If I google `"cute-headers"` (with quotes), it won't give me as much good information as it gives me when I google `"cute_headers"`.

Reply:
We have done this in the passed, and it matches the [docs](https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#should-reference-names-use---or-_)

> Recipes should stick to the original name of a library as much as possible.